### ISSUE TITLE
Deprecation messages with namer

### ISSUE THREAD

**Issue by fabienlem** [dustin10/VichUploaderBundle#801]
Hello!

I'm reporting a non blocker issue.

I'm using VichUploaderBundle 1.6.2 with Symfony 3.4.1 and I get some deprecation messages.

Here is a sample of my config:
```
vich_uploader:
    ...
    mappings:
        ...
        project_attachment:
            ...
            namer: vich_uploader.namer_origname
```

When using this statement in twig:
```
{{ vich_uploader_asset(attachment, 'file') }}
```

The following message occurs:

> User Deprecated: The "vich_uploader.namer_origname.project_attachment" service is private, getting it from the container is deprecated since Symfony 3.2 and will fail in 4.0. You should either make the service public, or stop using the container directly and use dependency injection instead.

As it's non blocker issue I don't really need help but I hope my feedback will help you to improve the bundle.

Regards,

Fabien

**Comment by garak** [comment]
Hi Fabien, we fixed such deprecations in 1.7 version.
Update the bundle and you should be fine

**Comment by fabienlem** [comment]
My bad!

I updated and everything is fine!
Thank you for your quick reply, you can close this issue.

Fabien
